### ISSUE TITLE
Trying to understand the forward function

### ISSUE THREAD

**Issue by hxu105** [DeepGraphLearning/torchdrug#154]
Hi, I would like to ask one question about the model implementation. I try to get a better understanding of the model behavior. I am using the ProtBert, and the forward function is in the following picture. 
<img width="734" alt="image" src="https://user-images.githubusercontent.com/40327643/201505204-fed45d0e-4b05-42ce-b7a6-068dcd0bf9d6.png">
The graph feature will be viewed as the loss value to do the backpropogation; however, I do not see any label involve here, so what is the meaning of this graph feature and why it can serves as the loss value. Any illustration will be appreciated!
Many thanks!

**Comment by Oxer11** [comment]
Hi!

The graph feature is the representations for proteins. The loss function is caluated in `tasks` module, e.g., [`tasks.PropertyPrediction`](https://torchdrug.ai/docs/api/tasks.html#propertyprediction).

**Comment by hxu105** [comment]
Ok, thanks for the answering!
